frontend/src/locales/es.yml

```
@@ -217,6 +217,20 @@ nav:
   loyalUsers: Fidelizados
   myOrders: Mis pedidos
   settings: Configuración
+  activity: Actividad
+
+activity:
```

YAMLlint (1.38.0)

[error] duplication of key "activity" in mapping

(key-duplicates)